test/resolvers/TestPublicResolver.js

```diff
@@ -11,18 +17,37 @@ const { exceptions } = require('../test-utils')
 contract('PublicResolver', function (accounts) {
   let node
   let ens, resolver, nameWrapper
-  const EMPTY_ADDRESS = '0x0000000000000000000000000000000000000000'
+  let account
+  let signers
+  let result
 
   beforeEach(async () => {
+    signers = await ethers.getSigners()
+    account = await signers[0].getAddress()
     node = namehash.hash('eth')
     ens = await ENS.new()
     nameWrapper = await NameWrapper.new()
+
+    //setup reverse registrar
+
+    const ReverseRegistrar = await deploy('ReverseRegistrar', ens.address)
+
+    await ens.setSubnodeOwner(ROOT_NODE, labelhash('reverse'), account)
+    await ens.setSubnodeOwner(
+      namehash.hash('reverse'),
+      labelhash('addr'),
+      ReverseRegistrar.address,
+    )
+
     resolver = await PublicResolver.new(
       ens.address,
       nameWrapper.address,
       accounts[9], // trusted contract
-      EMPTY_ADDRESS,
+      EMPTY_ADDRESS, //ReverseRegistrar.address,
```

Comment:
Why is ReverseRegistrar an empty address? Can you not set `ReverseRegistrar.address` instead?